docs/components/board/_index.md

```diff
@@ -1518,7 +1583,7 @@ interrupt.add_callback(callback_queue)
 
 **Parameters:**
 
-- `callback` [(chan Tick)](https://go.dev/tour/concurrency/2): The channel to add as a listener for when the state of the GPIO pin this interrupt is [configured for](#digital_interrupts) changes between high and low.
+- `callback` ([chan](https://go.dev/tour/concurrency/2) [Tick](https://pkg.go.dev/go.viam.com/rdk/components/board#Tick)): The channel to add as a listener for when the state of the GPIO pin this interrupt is [configured for](#digital_interrupts) changes between high and low.
```

Comment:
just noticed this, can't comment on the actual lines but the Addcallback section above this can be removed as it won't be implemented anymore.

Reply:
gotcha! I'll remove in new PR